metrics/edfi_tech_metrics/ossf_score.py

```diff
@@ -37,7 +35,36 @@ def fetch_repositories(organization):
     response = requests.get(url)
     if response.status_code == 200:
         data = response.json()
-        repositories = [repo["full_name"] for repo in data]
+        repositories = [repo['full_name'] for repo in data]
         return repositories
     else:
         return []
+
+
+def compare_with_previous_scores(current_scores):
+    current_df = pd.DataFrame(list(current_scores.items()), columns=['Repository', 'Current Score'])
+
+    data_dir = "./data/ossf-scores/"
+    today = datetime.now().strftime("%Y-%m-%d")
+    csv_files = [f for f in os.listdir(data_dir) if f.endswith('.csv') and f != f"{today}.csv"]
+
+    if not csv_files:
+        print("No previous scores found for comparison.")
+        return current_df, [], []
+
+    latest_file = max(csv_files)
```

Comment:
This name confused me a little. In my mind, "latest" would be "today". Since the next line creates `previous_df`, perhaps it would make sense to rename the variable to `previous_file`?